Author: Julian Taylor <jtaylor.debian@googlemail.com>
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/ipython/+bug/241535
Subject: use LightBG colorscheme by default
 Linux scheme leads to invisible text when console background is light
 LightBG is at least readable with dark background

Index: trunk/IPython/core/interactiveshell.py
===================================================================
--- trunk.orig/IPython/core/interactiveshell.py	2011-06-23 10:55:25.000000000 +0200
+++ trunk/IPython/core/interactiveshell.py	2011-06-23 10:58:33.033373573 +0200
@@ -119,7 +119,7 @@
     elif os.name=='nt':
         return 'Linux'
     else:
-        return 'Linux'
+        return 'LightBG'
 
 
 class SeparateUnicode(Unicode):
